less/about/preferences.less

```diff
@@ -455,6 +455,46 @@ table.sortableTable {
   overflow-y: scroll;
 }
 
+.modal .dialog.paymentHistory .sectionTitle {
+  text-align: left;
+}
+
+span.paymentHistoryButton.browserButton {
```

Comment:
this should not overflow when the window is narrow:
![screen shot 2016-08-25 at 10 42 37 am](https://cloud.githubusercontent.com/assets/549654/17979679/ca1913b0-6ab0-11e6-8c1b-b3f6859a8616.png)